Change RTM tests to MOSART,fix long line,

fischer-ncar opened the pull request:
Change RTM tests to MOSART when CLM4.5 or newer is used.  Fix line
that was too long n shr_frz_mod.F90.in.  Removed use_init_interp for the
b1850 usermods for clm.

Test suite:  ERS.f09_g16.B1850RG.yellowstone_intel
Test baseline: N/A
Test namelist changes: RTM tests changed to MOSART for CLM4.5 or newer
Test status: bit for bit

Fixes: none

User interface changes?: No

Code review:

fischer-ncar committed:
Change RTM tests to MOSART,fix long line,

Change RTM tests to MOSART when CLM4.5 or newer is used.  Fix line
that was too long n shr_frz_mod.F90.in.  Removed use_init_interp for the
b1850 usermods for clm.

Test suite:  ERS.f09_g16.B1850RG.yellowstone_intel
Test baseline: N/A
Test namelist changes: RTM tests changed to MOSART for CLM4.5 or newer
Test status: bit for bit

Fixes: none

User interface changes?: No

Code review:

jedwards4b commented:
Note that the commit template is now provided for you in the PR.   I have reviewed these changes - ready to merge.

fischer-ncar committed:
Add 5 day debug prebeta smoke test for f09_g16 B1850

Test suite:
Test baseline:
Test namelist changes: Add SMS_D.f09_g16.B1850 prebeta test
Test status: bit for bit

Fixes: none

User interface changes?: none

Code review:
